### PR TITLE
[PLAYER-1678] Added remaining unit tests

### PR DESCRIPTION
**Pull request by jpfranco**
Apart from the unit tests, this PR also includes:

- Fixed a few React warnings
- Switched the remaining control bar buttons to `AccessibleButton`'s.

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: tests/components/colorSelector-test.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+          accentColor: 'blue'
+        }
+      }
+    };
+  });
+
+  it('should render a ColorSelector', function() {
+    renderComponent();
+    expect(component).toBeTruthy();
+    expect(buttons.length).toBe(6);
```
Comment (aeng7) [inline tests/components/colorSelector-test.js]: Should we compare this against colors.length rather than 6?

File: tests/components/slider-test.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    expect(element.getAttribute('aria-label')).toBe(props.ariaLabel);
+    expect(element.getAttribute('aria-valuemin')).toBe('0');
+    expect(element.getAttribute('aria-valuemax')).toBe('100');
+    expect(element.getAttribute('aria-valuenow')).toBe('50');
+    expect(element.getAttribute('aria-valuetext')).toBe('50% ' + props.settingName);
+  });
+
+  it('should set a default focus id if none is provided', function() {
+    renderComponent();
+    expect(element.getAttribute(CONSTANTS.KEYBD_FOCUS_ID_ATTR).length).toBe(10);
```
Comment (aeng7) [inline tests/components/slider-test.js]: Where does the number 10 come from?

File: tests/components/slider-test.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  });
+
+  it('should update ARIA values when value changes', function() {
```
Comment (aeng7) [inline tests/components/slider-test.js]: Should we add some tests for negative numbers and numbers greater than 1?

File: tests/components/colorSelector-test.js (hunk 1 above)
Comment (jpfranco) [reply tests/components/colorSelector-test.js]: Updating

File: tests/components/slider-test.js (hunk 2 above)
Comment (jpfranco) [reply tests/components/slider-test.js]: This is just the length of the generated strings. Since they're random I can't check the actual value, but this is a bit better than checking that it's truthy.

**Comment by aeng7** [comment]
LGTM outside a few minor comments

File: tests/components/slider-test.js (hunk 2 above)
Comment (aeng7) [reply tests/components/slider-test.js]: Sounds good

**Comment by aeng7** [comment]
Feel free to merge after updating
